packages/core/src/indexing/build-indexing.ts

```diff
@@ -53,6 +54,14 @@ export function buildIndexing(indexingConfig: IndexingConfig): DatabaseIndexApi
       })
       return new SqliteIndexApi(dbConnection, indexingConfig.models)
     }
+    case 'postgres': {
+      const dataSource = knex({
+        client: 'pg',
+        connection: connectionString.toString(),
+        searchPath: ['ceramic', 'public'],
```

Comment:
@willex you sure we need this flexibility?

Reply:
yeah, this can be removed.